### ISSUE TITLE
[stable/concourse] web container is ignoring POSTGRES_HOST env variable

### ISSUE THREAD

**Issue by jsnider-mtu** [helm/charts#8710]
**Is this a request for help?**:
No
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
Bug Report

**Version of Helm and Kubernetes**:
Client: &version.Version{SemVer:"v2.10.0", GitCommit:"9ad53aac42165a5fadc6c87be0dea6b115f93090", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.11.0", GitCommit:"2e55dbe1fdb5fdb96b75ff144a339489417b146b", GitTreeState:"clean"}
Kubernetes v1.11.2 on AKS

**Which chart**:
stable/concourse

**What happened**:
The web container tries to connect to the PostgreSQL database using the host 127.0.0.1 even though I've set concourse.web.postgres.host to another name.
The POSTGRES_HOST env variable is set correctly in the container but the logs show that it is trying to connect to 127.0.0.1:5432 regardless.

**What you expected to happen**:
Database connection is made to the configured host.

**How to reproduce it** (as minimally and precisely as possible):
helm install stable/concourse --set concourse.web.postgres.host=concourse-release-postgresql --name concourse-release --namespace concourse

**Anything else we need to know**:
If I do an install using all default values then it uses the clusterIP of the postgresql service

**Comment by mbraku** [comment]
Hi, I ran into the same issue today. I think it has something to do with the fact that Concourse accidentally removed support for the `CONCOURSE_POSTGRES_DATA_SOURCE` environment variable in 4.1.0.

To fix it, I ran `helm repo update` (and pegged the chart version to the latest version I found on https://kubernetes-charts.storage.googleapis.com/, which was `2.0.2`). 

With that new version, the new environment variables `CONCOURSE_POSTGRES_*` are set instead of that deprecated variable. Now everything seems to work for us. Hope this helps!
